pkg/virt-controller/services/renderresources.go

```diff
@@ -199,7 +199,7 @@ func WithAutoMemoryLimits(namespace string, namespaceStore cache.Store) Resource
 	}
 }
 
-func WithCPUPinning(cpu *v1.CPU, cpuManagerPolicyBetaOption v1.CPUManagerPolicyBetaOptions) ResourceRendererOption {
+func WithCPUPinning(cpu *v1.CPU, annotations map[string]string) ResourceRendererOption {
```

Comment:
Commit message: Please consider adding a sentence about removing the annotation's value in order to align with the proposal.

Reply:
I don't want to reference the proposal if not necessary - IMO the commits should stand on their own right as much as possible. 
Is the reasoning here not enough?

Reply:
The next commit changes the annotation's name in order to align with the design, so I guess the removal of value is made for the same reason.